pkgs/development/libraries/libei/default.nix

```diff
@@ -0,0 +1,53 @@
+{ attr
+, fetchFromGitHub
+, fetchFromGitLab
+, lib
+, libevdev
+, libxkbcommon
+, meson
+, ninja
+, pkg-config
+, protobuf
+, protobufc
+, python3
+, python3Packages
+, stdenv
+, systemd
+}:
+let
+  munit = fetchFromGitHub {
+    owner = "nemequ";
+    repo = "munit";
+    rev = "fbbdf1467eb0d04a6ee465def2e529e4c87f2118";
+    hash = "sha256-qm30C++rpLtxBhOABBzo+6WILSpKz2ibvUvoe8ku4ow=";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "libei";
+  version = "1.0.0rc1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "libinput";
+    repo = "libei";
+    rev = "0.99.1";
```

Comment:
I was considering everybody everywhere would use the "released" instead of the tagged, but that [does not look true for FreeBSD Ports](https://www.freshports.org/x11/libei) -- which is the only one besides us releasing this now: https://repology.org/project/libei/versions

Reply:
Well, there are plausible reasons to FreeBSD doing this: they use a "specialized" fetcher for Github in a similar way we do: owner, repo, tag, hash. 
So it is easier to use the real tag instead of a "fantasy name".

(I remember a long time ago Openoffice did something similar: beta-2.0 = 1.99.99)

That being said, I opt to the real tag instead of fantasy version.